project/src/mocks/comments.ts

```diff
@@ -0,0 +1,17 @@
+import { CommentType } from '../types/comment-type';
+
+const comment = {
+  id: 1,
+  user: {
+    id: 12,
+    isPro: true,
+    name: 'Isaac',
+    avatarUrl: 'https://10.react.pages.academy/static/avatar/3.jpg',
+  },
+  rating: 4,
+  comment:
+    'We loved it so much, the house, the veiw, the location just great.. Highly reccomend :)',
+  date: '2022-06-13T12:25:36.938Z',
+};
+
+export const comments: CommentType[] = [comment];
```

Comment:
```suggestion
export const comments: CommentType[] = Array.from({length:5}, (_,id)({...comment, id}))
```
таким образом ты изготовишь 5 штук, похожих друг на друга, но с разными id и в специальное свойство key можешь подавать id - чтобы не забыть, что index (там) не следует использовать